### PR TITLE
Add until date to formereditor

### PR DESCRIPTION
**Pull request by fycdev**
**In Progress**

Add the ability to render a `retiredDate` property. In addition, if an editor has a `retiredDate` it is moved to the `formerEditors` list.

Still needs:
- [x]  testing
- [x] localisation
- [x] documentation in wiki.

### CODE REVIEW

File: src/w3c/headers.js [hunk 1]
```diff
@@ -395,18 +395,22 @@ export function run(conf) {
       }
     }
   };
+  if (!conf.formerEditors) conf.formerEditors = [];
   if (conf.editors) {
     conf.editors.forEach(peopCheck);
+    conf.formerEditors = conf.editors.reduce((collector, person) => {
```
Comment (marcoscaceres) [inline src/w3c/headers.js]: Probably want a comment here about what this is actually doing... relocating editor to former editor, iff `retiredDate` member is present.

File: examples/basic.html [hunk 2]
```diff
@@ -15,6 +15,12 @@
         editors: [{
           name: "Your Name",
           url: "https://your-site.com",
+          note: "blah"
```
Comment (marcoscaceres) [inline examples/basic.html]: Once this is all working, please revert changes to basic.html

**Comment by marcoscaceres** [comment]
Looking good. We'll need to add some tests :)

File: examples/basic.html (hunk 2 above)
Comment (fycdev) [reply examples/basic.html]: Would you like to include `retiredDate` example in `basic.html`?

Comment (marcoscaceres) [reply examples/basic.html]: Probably not. But we need to remember to add it to the wiki when we are ready to land this.

**Pull request merged**